pandas/tests/reshape/concat/test_categorical.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
+import pytest
+
+from pandas.core.dtypes.dtypes import CategoricalDtype
+
+import pandas as pd
+from pandas import Categorical, DataFrame, Series
+import pandas._testing as tm
+
+
+@pytest.fixture(params=[True, False])
```

Comment:
i would move fixtures to a concat/conftest.py (unless they are truly specific to a particular file)